src/components/subscriptions/SubscriptionData.jsx

```diff
@@ -1,8 +1,8 @@
 import React, { createContext, useMemo } from 'react';
 import PropTypes from 'prop-types';
+import { useSubscriptionData } from './data/hooks';
```

Comment:
tiny nit: Should this import be grouped with the external package imports, or stay with the relative import below?